src/app/_constants/paths.ts

```diff
@@ -98,7 +98,7 @@ export const PATHS = {
     {
       hasSubpaths: true,
       includesEntries: true,
-      customEntriesContentPath: '/ecosystem-explorer/projects/',
+      customEntriesContentPath: '/ecosystem-explorer',
```

Comment:
I believe we now don't need `customEntriesContentPath` anymore? Neither here nor anywhere else.

Reply:
Correct! Removed it ✅